woocommerce/woocommerce
# Fixed Webhooks and REST API Keys list table and settings

### All Submissions:






### Changes proposed in this Pull Request:



Fixes several bugs caused by changing the tab name from `api` to `advanced`, also change the form method from `GET` to `POST` and fix several notices in Webhooks and REST API keys settings.

Closes #20705.

### How to test the changes in this Pull Request:

1. Create webhooks and keys.
2. Try search, not that send a post and not display the "Your settings have been saved." notice.
3. Use bulk actions for both and check notices.

### Other information:




### Changelog entry

> Fixed search in Webhooks and REST API keys settings.
> Fixed REST API keys settings notices.
